src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -97,17 +99,23 @@ trait Backend {
   @throws[IllegalArgumentException]("if a value is missing / incorrect")
   def assertWorkflowOptions(options: WorkflowOptions): Unit = {}
 
-  def makeTag(backendCall: BackendCall): String = {
-    // Sometimes the class name is `anon$1`.  In cases like that, don't print it in the log because it's not adding value
-    val cls = this.getClass.getSimpleName
-    val clsString = if (cls.startsWith("anon")) "" else s"$cls "
-    s"$clsString[UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.key.tag}]"
+  private def backendClass: String = backendType match {
+    case BackendType.LOCAL => "LocalBackend"
+    case BackendType.JES => "JesBackend"
+    case BackendType.SGE => "SgeBackend"
+    case o => o.toString
```

Comment:
Well actually if it's just a String for logging, then maybe nm.  It would probably annoy a developer quickly enough to get fixed.  :smile: